back/engines/commercial/analytics/spec/acceptance/analytics_visits_spec.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+# Tests for queries needed by the visitors dashboard
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Analytics API - Queries for visitors dashboard', use_transactional_fixtures: false do
+  post 'web_api/v1/analytics' do
+    before do
+      header 'Content-Type', 'application/json'
+      admin_header_token
+
+      # Create 3 visits - 2 visitors
+      # Visitor 1 - 2 visits, no user, with a project, in Sept
+      create(:project)
+      locale1 = create(:dimension_locale_en)
+      visit1 = create(:fact_visit, matomo_visit_id: 1, duration: 100, pages_visited: 1)
+      visit1.dimension_projects << Analytics::DimensionProject.first
+      visit1.dimension_locales << locale1
+      visit2 = create(:fact_visit, matomo_visit_id: 2, duration: 200, pages_visited: 2)
+      visit2.dimension_projects << Analytics::DimensionProject.first
+      visit2.dimension_locales << locale1
+
+      # Visitor 2 - 1 visit, admin user, same channel, no project, different locale, in Aug
+      create(:user, roles: [{ type: 'admin' }])
+      locale2 = create(:dimension_locale_nl)
+      aug = create(:dimension_date_aug)
+      visit3 = create(
+        :fact_visit,
+        matomo_visit_id: 3,
+        duration: 300,
+        pages_visited: 3,
+        visitor_id: 'YYYY1',
+        dimension_user: Analytics::DimensionUser.first,
+        dimension_date_first_action: aug,
+        dimension_date_last_action: aug
+      )
+      visit3.dimension_locales << locale2
+    end
+
+    it 'Gets correct number of visits & visitors and correctly averages duration and pages visited' do
```

Comment:
Examples by convention starts by a small letter.
```suggestion
    it 'gets correct number of visits & visitors and correctly averages duration and pages visited' do
```